starlette_context/middleware/context_middleware.py

```diff
@@ -22,12 +22,18 @@ class ContextMiddleware(BaseHTTPMiddleware):
     """
 
     def __init__(
-        self, plugins: Optional[Sequence[Plugin]] = None, *args, **kwargs
+        self,
+        plugins: Optional[Sequence[Plugin]] = None,
+        defaut_error_response: Response = Response(status_code=400),
```

Comment:
How is this problem solved in other middlewares?

I think this parameter should be a callback function instead. Maybe accept `Exception` and `Request` and based on these two compose error message? From the other hand I don't think it's a good idea at all because there already is exception handler that does exactly that, but not for 5xx. 🤔